ospfd/ospfd.h

```diff
@@ -84,6 +84,9 @@ struct ospf_external {
 	struct route_table *external_info;
 };
 
+/* Status of various sub function e.g. TE */
+typedef enum _status_t {disable, enable} status_t;
```

Comment:
same thing as with the `opcode_t` enum that @qlyoung commented on -- but in this case
I would suggest to

* remove the enum
* change the field in `struct ospf_pce_info` from
  `status_t status;`
  to ->
  `bool enabled;`
* replace all `if (xxxxx->status == enable)` with `if (xxxxx->enabled)`